### PR TITLE
Messages

### PR DESCRIPTION
**Pull request by or-else**
Refactoring Find view and New Chats view to use common table cells + some other UI tweaks.

### CODE REVIEW

File: Tinodios/FindViewController.swift [hunk 1]
```diff
@@ -181,3 +171,17 @@ extension FindViewController: UISearchResultsUpdating, UISearchControllerDelegat
...
+    func selected(from cell: UITableViewCell) {
+        guard let indexPath = tableView.indexPathForSelectedRow else { return }
+        guard let id = getUniqueId(for: indexPath) else { return }
+        // If the search bar is active, deactivate it.
+        if searchController.isActive {
+            DispatchQueue.main.async {
+                self.searchController.isActive = false
+            }
+        }
+        presentChatReplacingCurrentVC(with: id)
```
Comment (aforge) [inline Tinodios/FindViewController.swift]: The problem is it doesn't work when you type something in the search bar:
self.searchController.isActive = false makes the view reload the list of contacts and the presentChatReplacingCurrentVC() call fails for some reason.
Maybe try to not trigger a reload when cancelling the search bar in this specific case?

File: Tinodios/NewGroupViewController.swift [hunk 2]
```diff
@@ -98,13 +109,31 @@ class NewGroupViewController: UIViewController, UITableViewDataSource {
         let tagsList = Utils.parseTags(from: tags)
         self.interactor?.createGroupTopic(titled: groupName, subtitled: privateInfo, with: tagsList, consistingOf: members, withAvatar: avatar)
     }
+
+    /// Show message that no members are selected.
+    public func toggleNoSelectedMembersNote(on show: Bool) {
```
Comment (aforge) [inline Tinodios/NewGroupViewController.swift]: Can it be private?

File: Tinodios/NewGroupViewController.swift [hunk 3]
```diff
@@ -98,13 +109,31 @@ class NewGroupViewController: UIViewController, UITableViewDataSource {
...
+        if show {
+            let messageLabel = UILabel(frame: CGRect(x: 0, y: 0, width: selectedCollectionView.bounds.size.width, height: selectedCollectionView.bounds.size.height))
+            messageLabel.text = "No members selected"
+            messageLabel.textColor = .gray
+            messageLabel.numberOfLines = 0;
```
Comment (aforge) [inline Tinodios/NewGroupViewController.swift]: No need for semicolons (it's Swift) ;-)

File: Tinodios/widgets/RoundImageView.swift [hunk 4]
```diff
@@ -75,21 +100,36 @@ class RoundImageView: UIImageView {
...
         } else {
-            self.letterTileFont = UIFont.preferredFont(forTextStyle: .title2)
-            let (fg, bg) = RoundImageView.selectBackground(id: id, dark: Tinode.topicTypeByName(name: id) == TopicType.p2p)
-            self.letterTileTextColor = fg
-            self.backgroundColor = bg
-
-            // Avatar placeholder.
-            var firstChar = title ?? ""
-            firstChar = firstChar.isEmpty ? "?" : firstChar
+            if id != nil && !id!.isEmpty {
```
Comment (aforge) [inline Tinodios/widgets/RoundImageView.swift]: A more swifty way would be
if let id = id, !id.isEmpty {}

Comment (or-else) [reply Tinodios/widgets/RoundImageView.swift]: done

File: Tinodios/NewGroupViewController.swift (hunk 3 above)
Comment (or-else) [reply Tinodios/NewGroupViewController.swift]: done

File: Tinodios/NewGroupViewController.swift (hunk 2 above)
Comment (or-else) [reply Tinodios/NewGroupViewController.swift]: done

File: Tinodios/FindViewController.swift (hunk 1 above)
Comment (or-else) [reply Tinodios/FindViewController.swift]: fix

**Pull request merged**